website/docs/docs/collaborate/govern/model-versions.md

```diff
@@ -6,39 +6,140 @@ description: "Version models to help with lifecycle management"
+
+| v | version    | `ref` syntax                                          | File name                                       | Database relation                                                        |
+|---|------------|-------------------------------------------------------|-------------------------------------------------|--------------------------------------------------------------------------|
+| 3 | "prerelease" | `ref('dim_customers', v=3)`                           | `dim_customers_v3.sql`                          | `analytics.dim_customers_v3`                                             |
+| 2 | "latest"     | `ref('dim_customers', v=2)` **and** `ref('dim_customers')`  | `dim_customers_v2.sql` **or** `dim_customers.sql` | `analytics.dim_customers_v2` **and** `analytics.dim_customers` (recommended) |
+| 1 | "old"        |  `ref('dim_customers', v=1)`                           | `dim_customers_v1.sql`                          | `analytics.dim_customers_v1`                                             |
+
+As you'll see in the implementation section below, a versioned model can reuse the majority of its yaml properties and configuration. Each version needs to only say how it _differs_ from the shared set of attributes. This gives you, as the producer of a versioned model, the opportunity to highlight the differences across versions—which is otherwise difficult to detect in models with dozens or hundreds of columns—and to clearly track, in one place, all versions of the model which are currently live.
+
+dbt also supports [`version`-based selection](node-selection/methods#the-version-method). For example, you could define a [default yaml selector](node-selection/yaml-selectors#default) that avoids running any old model versions in development, even while you continue to run them in production through a sunset and migration period. (You could accomplish something similar by applying `tags` to these models, and cycling through those tags over time.)
+
+<File name="selectors.yml">
+
+```yml
+selectors:
+  - name: exclude_old_versions
+    default: "{{ target.name == 'dev' }}"
+    definition:
+      method: fqn
+      value: "*"
+      exclude:
+        - method: version
+          value: old
+```
+
+</File>
+
+Because dbt knows that these models are _actually the same model_, it can notify downstream consumers as new versions become available, and (in the future) as older versions are slated for deprecation.
+
+```bash
+Found an unpinned reference to versioned model 'dim_customers'.
+Resolving to latest version: my_model.v2
+A prerelease version 3 is available. It has not yet been marked 'latest' by its maintainer.
+When that happens, this reference will resolve to my_model.v3 instead.
+
+  Try out v3: {{ ref('my_dbt_project', 'my_model', v='3') }}
+  Pin to  v2: {{ ref('my_dbt_project', 'my_model', v='2') }}
+```
 
 ## How to create a new version of a model
```

Comment:
Not for this week, but eventually, we might want to pull the procedure section out into its own page so people who don't need the "why" and other context can more easily find the "how."

Reply:
Heard! A lot of the "how" details are also captured in the reference documentation: https://docs.getdbt.com/reference/resource-properties/versions

For now, we should expect that the main audience for these docs is people trying to learn about & understand the new feature. As more the concept becomes more established, and people are already convinced they want to use the feature, we can make it even easier to _just do the thing_